deps.edn

```diff
@@ -451,7 +451,9 @@
                                  ;; too many false positives for now
                                  :unused-ret-vals
                                  ;; Kondo lints this for us anyway, and this isn't as easy to configure.
-                                 :wrong-arity]
+                                 :wrong-arity
+                                 ;; Kondo lints this for us anyway, and this isn't as easy to configure.
+                                 :suspicious-expression]
```

Comment:
disable this because the change to the match started triggering this and eastwood is supplanted by kondo anyways.